src/components/style/forms/iconInput.css

```diff
@@ -0,0 +1,29 @@
+
+.fdsIconInput {
+  position: relative;
+  width: min-content;
+}
+
+.fdsIconInput--hasStartIcon input {
+  padding-left: var(--space-double);
+}
+
+.fdsIconInput--hasEndIcon input {
+  padding-right: var(--space-double);
+}
+
+.fdsIconInput-icon {
+  position: absolute;
+  top: 1px;
+  bottom: 1px;
+  display: flex;
+  align-items: center;
```

Comment:
In my own components, I've been using an extra div to utilize `alignChild--center--center`. This is more of a lesson to me, but it seems you're ok curving that class and spelling it out directly here?

Reply:
good call - I'll use the utility